data/en/privacy.yml

```diff
@@ -3,6 +3,10 @@ privacy:
   tagline: OUR PRIORITY
   title: PRIVACY & SECURITY
   description: Rotational is deeply committed to privacy and security. AI offers tremendous potential benefit to humanity, provided it is used properly, and enterprises commit to being good data stewards. To that end, we have committed to the US-EU Data Privacy Framework (DPF) and prioritize privacy and security in all our product and services work.
+  image: img/security_and_privacy.webp
```

Comment:
The image source was moved to the data file for consistency and make it easier to update content.